docker/flower-waits-for-cachito.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
+
+until curl --fail --silent --show-error http://cachito-api:8080/api/v1/status/short; do
```

Comment:
Should this be parametrized somehow to make it easier to deploy it in other environments? For instance, the base URL could be passed as a param to this script. Same goes for all the ports. Otherwise, we would need a copy of this script for every environment (this could also be acceptable though, but then the script would have to be mounted in the container).

Reply:
@athos-ribeiro Ok, there are new parameters for hosts/ports/creds